.gitignore

```diff
@@ -1,3 +1,4 @@
+/shallowHRD
```

Comment:
Why is this needed?

Reply:
It's not -- I think this is a relic from many commits ago when Carlos was working on the package.